src/include/kdb.h.in

```diff
@@ -64,12 +69,15 @@ enum {
 	KEY_META_NAME=1<<21,
```

Comment:
Maybe a stupid comment, but this could strictly speaking overflow the int type, right? Comment is unrelated to this PR.

Reply:
No stupid comment at all. There are several places where int cannot be 2 bytes but must be 4. This is only documented in kdbos.h, quite hidden.

A solution that would avoid this problem would be to include kdbtypes.h in kdb.h and avoid int, size_t and ssize_t but this would be a big effort...

Reply:
I was unaware of this. I think it is not a high priority for now but good to keep in mind that the problem exists. Thanks for the clarification.